tools/codeformat.py

```diff
@@ -105,12 +110,58 @@ def fixup_c(filename):
         assert not dedent_stack, filename
 
 
+def query_git_files(verbose):
+    def list_results(cmd):
```

Comment:
Maybe call it `cmd_result_set()`, to indicate it returns a set not a list?

Reply:
Ah yes, it was previously a list... then I had some duplicates...